docs/ConverterUsage.md

```diff
@@ -0,0 +1,98 @@
+# Converter Usage & Demo
+
+## Usage
+
+`jfrconv [options] <input> [<input>...] <output>`
+
+The output format specified can be only one at a time for conversion from one format to another.
+
+### Available arguments
+
+```
+Conversion options: 
+  -o --output FORMAT   Output format: <filename> followed by any of the extensions: html, collapsed, pprof, pb.gz
```

Comment:
No, `-o` specifies format, not a filename, e.g. `-o collapsed`.
`-o` can be omitted, if the output file extension unambiguously determines the format.

For the doc, it makes sense to expand the meaning of different formats: flame graph, collapsed stack traces (with a reference), Google pprof (with a reference) and compressed pprof.